### PR TITLE
Add & apply pre-commit and linting

### PR DESCRIPTION
**Pull request by jwodder**
Part of #325.

### CODE REVIEW

**Comment by jwodder** [comment]
@yarikoptic I expect there will be more PRs after this that will need to be merged quickly lest they create merge conflicts with other PRs created around the same time.  How should this be handled?

File: nidm/terms/imports/qibo_import.ttl [hunk 1]
```diff
@@ -39,35 +39,34 @@ rdf:type rdf:type owl:AnnotationProperty .
...
 qibo:Phantom_experimental_subject rdf:type owl:Class ;
-                                  
+
                                   rdfs:label "Phantom Experimental Subject"^^xsd:string  .
 
 
 
 
 ###  Generated by the OWL API (version 3.5.1) http://owlapi.sourceforge.net
-
```
Comment (yarikoptic) [inline nidm/terms/imports/qibo_import.ttl:73]: I would have left autogenerated files without changes -- not familiar yet with turtle format intricately but may be those space indentations are important part of the syntax...?  I have tested this one (with trailing spaces removed) at some randomly found http://ttl.summerofcode.be/ -- seems to be ok.  I think we can proceed but might need to be ready to revert this part of cleansing

**Comment by yarikoptic** [comment]
sorry for the delay. I left a comment, let's proceed

**Comment by yarikoptic** [comment]
> @yarikoptic I expect there will be more PRs after this that will need to be merged quickly lest they create merge conflicts with other PRs created around the same time. How should this be handled?

I will keep an eye.  Worse comes to worse, feel free to merge yourself, I will review later or just build up the chain of PRs basing one on top of another.

**Pull request merged**